docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       - ~/.ssh/known_hosts:/home/orise/.ssh/known_hosts:rw
       - ~/.ssh/id_rsa:/home/orise/.ssh/id_rsa:ro
       - ~/.ssh/id_rsa.pub:/home/orise/.ssh/id_rsa.pub:ro
+      - ~/.gnupg:/home/orise/.gnupg:rw
```

Comment:
I confess I don't like the idea of exposing all my secret keys just for the sake of git's `user.signingKey`. Shouldn't be the user the only one responsible for doing such thing?

Maybe we could allow the user choose which volumes are going to be shared with the container.

Just a final question, why rw access here?

Reply:
For the `test` service, shouldn't we also expose the gnupg folder since we are already exposing the gitconfig?

Reply:
> I confess I don't like the idea of exposing all my secret keys just for the sake of git's user.signingKey

I agree and already have created issue #38 to address this problem using docker secrets. I didn't try this because I think it requires some investigation and I wanted a quick solution to it

> Shouldn't be the user the only one responsible for doing such thing?
>
> Maybe we could allow the user choose which volumes are going to be shared with the container.

It sounds good. I am only wondering if it is not too specific. Maybe it is better to let it entirely to the user. However, I didn't find a smooth way of doing this other than sharing the volume as I did.

> Just a final question, why rw access here?
Good question. I think I set it as rw to allow the user to manage the keys from inside the container.

Reply:
> For the test service, shouldn't we also expose the gnupg folder since we are already exposing the gitconfig?

I didn't add to the test image because I see it is not the purpose of the image to make commits and pushes inside it. The gitconfig is needed to be able to download private repositories.